cmake/PrintUsefulCMakeInfo.cmake

```diff
@@ -71,3 +71,11 @@ if(DOXYGEN_FOUND)
 else()
   message(STATUS "Doxygen: Not found, documentation cannot be built.")
 endif()
+
+if(BUILD_PYTHON_BINDINGS AND "${JEMALLOC_LIB_TYPE}" STREQUAL SHARED)
+  message(STATUS
+    "When using the python bindings you must run python as:\n"
+    "   LD_PRELOAD=${JEMALLOC_LIBRARIES} python ..."
```

Comment:
Please add `\n` at the end of this line

Reply:
Okay, done and rebased. Added to the next line too to be consistent and keep messages from getting too long on a single line.